src/_locales/bg/messages.json

```diff
@@ -1,6 +1,6 @@
 {
 	"appDesc": {
-		"message": "Записвайте всичко от уеб в OneNote. Изрежете го в OneNote, организирайте го и го редактирайте, след което ще имате достъп до него от всяко устройство.",
+		"message": "Запазете уеб съдържание в OneNote. Изрязвайте, организирайте, редактирайте и достъпвайте от всяко устройство.",
```

Comment:
lets a open a tracking bug for localization for all locales using our pipelines.

Reply:
Created the following tracking bug:

[Bug 9655291](https://office.visualstudio.com/OneNote/_workitems/edit/9655291): [OneNote][Online][Integration]: Localization support for all locales in OneNote web clipper